lke_node_pools.go

```diff
@@ -163,6 +174,28 @@ func (c *Client) DeleteLKENodePool(ctx context.Context, clusterID, poolID int) e
 	return err
 }
 
+// GetLKENodePoolNode gets the LKENodePoolLinode with the provided ID
+func (c *Client) GetLKENodePoolNode(ctx context.Context, clusterID int, nodeID string) (*LKENodePoolLinode, error) {
+	e := formatAPIPath("lke/clusters/%d/nodes/%s", clusterID, nodeID)
+	response, err := doGETRequest[LKENodePoolLinode](ctx, c, e)
+	if err != nil {
+		return nil, err
+	}
+
+	return response, nil
+}
+
+// RecycleLKENodePoolNode recycles a LKENodePoolLinode
+func (c *Client) RecycleLKENodePoolNode(ctx context.Context, clusterID int, nodeID string) error {
+	e := formatAPIPath("lke/clusters/%d/nodes/%s/recycle", clusterID, nodeID)
+	_, err := doPOSTRequest[LKENodePoolLinode, any](ctx, c, e)
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
Maybe return the `*LKENodePoolLinode` here?
```suggestion
func (c *Client) RecycleLKENodePoolNode(ctx context.Context, clusterID int, nodeID string) (*LKENodePoolLinode, error) {
	e := formatAPIPath("lke/clusters/%d/nodes/%s/recycle", clusterID, nodeID)
	return doPOSTRequest[LKENodePoolLinode, any](ctx, c, e)
```

Reply:
I don't think the endpoint returns the node according to the API Techdocs: https://techdocs.akamai.com/linode-api/reference/post-lke-cluster-node-recycle

Reply:
Yes, exactly, I just learned that as well lol